### PR TITLE
Test: implemented model upload tests.

### PR DESCRIPTION
**Pull request by Leon-Leyang**
## Issue ticket number and link

Issue #186 

## What are the changes

Test: implemented model upload tests.

## Comments

--

## Checklist before requesting a review

- [x] Double check the diff between my branch and the target branch. Make sure there's nothing unexpected.
- [x] Tag relevant issues and PRs in the first section of this form.
- [x] All CI tests are passed.
- [x] No `package-lock.json` or `package.json` changes are pushed. If you do, add a new section `Dependency Changes` to the form stating what library changes are introduced and why.

### CODE REVIEW

File: database/scripts/upload-model.py
Comment (Leon-Leyang) [inline database/scripts/upload-model.py:1]: I guess we can remove the whole /database right now?

Comment (PaulCCCCCCH) [reply database/scripts/upload-model.py:1]: Yes, code in that folder is no longer used. Let's remove it in another PR.

File: back-end/tests/test_model.py [hunk 1]
```diff
@@ -61,10 +277,21 @@ def dummy_api_delete_model(client, name: str):
 
 class TestModel:
     class TestModelUpload:
-        def test_model_upload(self, client, reset_db):
-            # TODO:
-            response = dummy_api_upload_dummy_model(client, "test-resnet-18")
-            assert response.status_code == 200
+        @pytest.mark.parametrize("test_data", upload_test_cases)
```
Comment (PaulCCCCCCH) [inline back-end/tests/test_model.py:280]: This is so much more clear! Thanks!

File: back-end/tests/test_edit.py [hunk 2]
```diff
@@ -56,7 +56,9 @@ def test_user_edit_fail_broken_image_data(self, client, reset_db):
             assert rv["error_code"] == -1
             assert rv["detail"] == "Broken image, fail to decode"
 
-    def test_user_edit_success(self, client, reset_db):  # TODO: fail due to inactive database
+    def test_user_edit_success(
+        self, client, reset_db
+    ):  # TODO: fail due to inactive database
```
Comment (PaulCCCCCCH) [inline back-end/tests/test_edit.py]: I think this problem has been fixed. Shall we remove this TODO?

**Review by PaulCCCCCCH — APPROVED** [review]
Thanks a lot! Overall looks good!

File: database/scripts/upload-model.py
Comment (Leon-Leyang) [reply database/scripts/upload-model.py:1]: Got it~

**Pull request merged**